one-offs/initial_nvidia_config_and_disable_cgroups_v2_playbook.yml

```diff
@@ -0,0 +1,30 @@
+- hosts: all
```

Comment:
hosts: all?

Reply:
intention is to run with a `--limit` for testing, but should be defined properly, as you say.